src/runtime/policy/tests/policy-test.ts

```diff
@@ -808,9 +808,97 @@ policy MyPolicy {
          '',
          /* canWriteSuperset = */manifestPerson,
          /* canReadSubset = */personSubset);
-       assert.deepEqual(ingressValidation.getMaxReadType(subsetTypeVar), subsetTypeVar);
+       const expectedTypeVar = TypeVariable.make(
+         '',
+         /* canWriteSuperset = */personSubset,
+         /* canReadSubset = */personSubset);
+       assert.deepEqual(
+         ingressValidation.getMaxReadType(subsetTypeVar), expectedTypeVar);
      });
 
+  it('updates write w/ max read type that is consistent with read', async() => {
```

Comment:
there are 3 possible cases in the updated method above:
```
if (maxReadType.isAtLeastAsSpecificAs(canReadSubset)) {
    newCanWriteSuperset = canReadSubset;
} else if (canReadSubset.isAtLeastAsSpecificAs(maxReadType)) {
    newCanWriteSuperset = maxReadType;
} else {
    return TypeVariable.make(...)
}
```
are there unit tests for each?

Reply:
Yes. The following tests cover the other cases:
 - https://github.com/PolymerLabs/arcs/blob/0af125651abc4e3bc1684522a366b27c5459f16f/src/runtime/policy/tests/policy-test.ts#L746
  - https://github.com/PolymerLabs/arcs/blob/0af125651abc4e3bc1684522a366b27c5459f16f/src/runtime/policy/tests/policy-test.ts#L778